config/config_test.go

```diff
@@ -70,6 +71,77 @@ func TestDefaultSampler(t *testing.T) {
 	require.Error(t, err)
 }
 
+func TestServiceNameFromEnv(t *testing.T) {
+	os.Setenv("JAEGER_SERVICE_NAME", "my-service")
```

Comment:
Shouldn't we use local constants for these ENV names?

Reply:
In Java, yes. In Go? I don't know... I have a feeling that creating a constant `JAEGER_SERVICE_NAME` with the value `JAEGER_SERVICE_NAME` is not idiomatic

Reply:
The constant name can be different, go idiomatic. I am asking for it because there is some repetion - code, tests, logs, comments

Reply:
yes, typically we try to avoid string constants directly in the code. Having them all in a single place as constants makes it easier to survey and document